ansible/vars/icds/icds_public.yml

```diff
@@ -0,0 +1,197 @@
+SITE_HOST: 'icds-cas.gov.in'
+NO_WWW_SITE_HOST: 'icds-cas.gov.in'
+deploy_env: icds
+
+riakcs_s3_access_key: "{{ secrets.RIAKCS_S3_ACCESS_KEY }}"
+riakcs_s3_secret_key: "{{ secrets.RIAKCS_S3_SECRET_KEY }}"
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: False
+  cchq_http: True
+  cchq_http_redirect: True
+  cchq_http_j2me: False
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: True
+  dhis: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech1: False
+  motech2: False
+  riakcs: True
+  cas_ssl: False
+
+proxy_type: 'nginx'
+primary_ssl_env: "icds"
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+fake_ssl_cert: no
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_cluster_name: 'prodhqes-1.x'
```

Comment:
don't think so - but we're going to be deleting all the data when we do the migration anyway